src/main/java/org/openstreetmap/josm/plugins/ods/bag/osm/build/BagBuildingEntityPrimitiveBuilder.java

```diff
@@ -37,7 +37,7 @@ protected void buildTags(OdBuilding building, Map<String, String> tags) {
         if (building.getStartDate() != null) {
             tags.put("start_date", building.getStartDate());
         }
-        if ("Sloopvergunning verleend".equals(building.getStatus())) {
+        if (building.getStatus() == EntityStatus.REMOVAL_DUE) {
             tags.put("note", "Sloopvergunning verleend");
```

Comment:
Deze note wordt nooit toegevoegd, omdat `"Sloopvergunning verleend"` nooit gelijk is aan `EntityStatus#REMOVAL_DUE`.